### DISCUSSION TITLE
The purpose of community.anytype.io

### DISCUSSION THREAD

**Discussion by vintprox** [anyproto/contributors#25] [Q&A] [answered]
To address #24, there are few questions nagging my mind with Anytype Community.

- What's the purpose of [the forum](https://community.anytype.io/) based on Discourse engine?
- How does it relate to [Discussions](https://github.com/orgs/anyproto/discussions)?
- Why can't I sign up there?

**Comment by fuksman** [comment, marked as answer]
Hey @vintprox,

- [forum](https://community.anytype.io/) is a users community, dedicated to questions about how to **use** Anytype
- [discussion](https://github.com/orgs/anyproto/discussions) is a contributors community, dedicated to questions about how to **build** Anytype, Any-Sync, and, in the coming future, use-cases, integrations, etc.
- registration on the forum is invite-only for now, you can find the link inside Anytype app, [here](https://doc.anytype.io/d/community/community-forum) are the details

**Reply by vintprox** [reply]
I see! To summarize in my words, forum is a bubble of users that have found their way to join discourse through the app, and Discussions is a meta community to discuss things under the microscope.

**Reply by developomp** [reply]
If GH discussions will be used for building anytype-related projects, I think it would make more sense to remove the [show-and-tell](https://github.com/orgs/anyproto/discussions/categories/show-and-tell) category in favor of [showcase](https://community.anytype.io/c/general-discussion/showcase) page.

**Reply by fuksman** [reply]
@developomp We've created it for some more hard-core showcases, like "Look, I've implemented an automation that takes all my tweets from Twitter export and stores them as objects inside Anytype space; let's make it a part of the apps" 
If it is empty, we'll close it.

**Reply by developomp** [reply]
I'm not sure if the intention will be picked up ealily by most people but sure.

**Reply by developomp** [reply]
Also, shouldn't we disable new submission for [forum feature request](https://community.anytype.io/c/feature-requests/6/none) and [forum bug report](https://community.anytype.io/c/bug-reports/7/none) now that we have GitHub?

**Reply by fuksman** [reply]
@developomp We're considering doing so in the future, as there are currently a lot of users who don't have a GitHub account. Additionally, creating bug reports and feature requests in GitHub requires an understanding of what causes the problem (client, middleware, infrastructure), which can be hard for a non-tech-savvy user. 
I also connect this question with the one about making our internal task tracker available for contributors somehow. So, our current goal is to design a combination of systems which will allow:
-  Posting issues for users, contributors, and Any QA team
-  Triaging issues to target repositories
-  Making this information available for users to upvote and for contributors and team members to prioritize and work on.

**Comment by derekbelrose** [comment]
I don't see a way to create an account on community.anytype.io, is this intentional?

**Reply by developomp** [reply]
I believe it is invitation only.

**Reply by fuksman** [reply]
Registration on the forum is invite-only for now, you can find the link inside Anytype app, [here](https://doc.anytype.io/d/community/community-forum) are the details

**Reply by derekbelrose** [reply]
Ok great thank you. Invite only isn't really following the open philosophy. Is there any plan to open it up to all users?

**Reply by fuksman** [reply]
It's open for all users, as invites are available to everyone who uses the app. Currently, it's the only method that has worked effectively in terms of spam protection.

**Reply by derekbelrose** [reply]
Ah, I see now.  Thank you for clarifying. I didn't realize the link in the app was an invitation instead of just a link to the community
